templates/communitypage.html

```diff
@@ -0,0 +1,161 @@
+        <h6 class="card-title">Following 1.2k</h6>
+        <button class="btn btn-outline-danger" style="border-radius:32px;">Join Community</button>
+      </div>
+    </div>
+  </div>
+  <div class="col-mb-4 mx-3">
+    <div class="card text-center" style="width: 20rem;">
+      <img src="https://source.unsplash.com/1600x900/?technology" class="card-img-top" alt="...">
+      <div class="card-body">
+        <h3 class="card-title">Technology</h3>
+        <h6 class="card-title">Following 1.2k</h6>
+        <button class="btn btn-outline-danger" style="border-radius:32px;">Join Community</button>
+      </div>
+    </div>
+  </div>
+</div>
+</div>
+<div class="container my-auto" style="text-align: center; padding-bottom: 30px;">
+  <hr>
+  <h4 id="subheading" style="text-align: center;">Subscribe to our newsletter to get latest blog post links</h4>
+  <div class="box" style="position: relative; margin-left: auto !important; margin-right: auto !important; padding-top: 20px;">
+      <input style = "height:40px; width:350px;" type="search" placeholder="Enter your mail id" aria-label="Search Communtiies">
+      <button class="btn btn-outline-warning my-2 my-sm-0" type="submit">Mail me</button>
+    </div>
+</div>
+<!-- Footer starts here -->
+<footer class="footer mt-auto py-3 bg-dark">
+  <div id="footer" class="container" style="color:white;">
+    <h3 id="footer-contents"> Blogger </h3>
+    <h4 id="footer-contents"> Basic Introduction</h4>
+  <div id="footer-contents">
+        <ul>
+          <li><a href="#" style="color:white;"><img src="https://img.icons8.com/fluent/40/000000/about.png"/> About</a></li>
+          <li><a href="#" style="color:white;" ><img src="https://img.icons8.com/flat_round/40/000000/home--v1.png"/> Home</a></li>
+          <li><a href="#" style="color:white;"><img src="https://img.icons8.com/color/40/000000/appointment-reminders.png"/> Notifications</a></li>
+        </ul>
+  </div>
+  <div id="footer-contents">
+        <ul>
+          <li><img src="https://img.icons8.com/fluent/32/000000/phone-disconnected.png"/> 1234567890</li>
+          <li><img src="https://img.icons8.com/nolan/32/user-location.png"/> Address</li>
+          <li><img src="https://img.icons8.com/fluent/32/000000/important-mail.png"/> Mail</li>
+        </ul> 
+  </div>
+  <div id="footer-contents">
+        <ul>
+          <li><a href="#"><img src="https://i.postimg.cc/7PTDC26P/facebook.png" alt="facebook" style="height:50px; width:50px;"/></a></li>
+          <li><a href="#" ><img src="https://i.postimg.cc/FdkwvJDQ/instagram.png" alt="instagram" style="height:50px; width:50px;"></a></li>
+          <li><a href="#" ><img src="https://i.postimg.cc/mDYwqjj2/linkedin.png" alt="linkedin" style="height:50px; width:50px;"></a></li>
+          <li><a href="#" ><img src="https://i.postimg.cc/QMR4vHrS/twitter.png" alt ="twitter" style="height:50px; width:50px;"></a></li>
+        </ul> 
+  </div>
+  </div>
+</footer>
+<!-- Footer ends here -->
```

Comment:
Let's keep the footer as well in a different file so that we can use it in other page as well.